deployments/data-hub/data-hub--staging.yaml

```diff
@@ -0,0 +1,230 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: data-hub--staging
```

Comment:
Just looking at the other charts in the repo, the current convention is `--stg` rather than `--staging`, so I think it would be best to aim for `data-hub--stg`. Can always change the naming in all the charts latter if you so desire.

Reply:
Done, thanks for spotting. I've looked at so many variations on "staging" I think my brain just filters them into the same words now :)